src/lib/seam/components/DeviceDetails/ThermostatDeviceDetails.tsx

```diff
@@ -106,6 +111,17 @@ export function ThermostatDeviceDetails({
                   temperatureUnit='fahrenheit'
                 />
               </DetailRow>
+              <DetailRow label={t.fanMode}>
+                <FanModeMenu
+                  mode={device.properties.fan_mode_setting ?? 'auto'}
```

Comment:
Why is the default auto? Is this just an issue with the types or can the fan mode really be undefined on the API?

Reply:
Ahh thank you—this is actually not required (forgot to remove this after resolved underlying type errors)